jitpack.yml

```diff
@@ -0,0 +1,6 @@
+jdk:
+  - openjdk11
+before_install:
+  - ./prepareEnvironment.sh
```

Comment:
What is this one doing?

Reply:
I used the [custom commands example](https://jitpack.io/docs/BUILDING/#custom-commands) as a template and assumed that `prepareEnvironment.sh` is required. But, in reality, only the `jdk` needs to be specified. I removed now the unnecessary lines.